gatorgrade/output/output_percentage_printing.py

```diff
@@ -0,0 +1,25 @@
+"""Output with the percentage of checks that the student has met requirments."""
+from colorama import Fore
+
+
+def print_percentage(results):
+    """Print percentage acts as fuction that will produce the output."""
+    # iterate through results tuples
+    true_list = []  # empty list for storing true results
+    for i in results:
+        for j in i:
```

Comment:
Once again you should look at the latest `out-feature-proper-checks-displayed` commit. There I refactored the code that you seem to have emulated here.  My new implementation simplifies the code by only using one for loop and indexing the result tuple to verify the check, since the boolean will always be in the same spot. If you used the result variable name instead of i, this would look like:
```python
if result[1] is True:
```

Reply:
Disregard that last comment, to stick with continuity I'm going to make the change suggested.